Tests/BaseTestCase.php

```diff
@@ -23,14 +23,15 @@ class BaseTestCase extends \PHPUnit_Framework_TestCase
 
     const SCHEMA_PRODUCT = 'Managlea\Tests\Models\Product';
 
+    /**
+     * @throws \Doctrine\ORM\Tools\ToolsException
+     */
     public function setUp()
     {
-        $isDevMode = true;
         $paths = array(__DIR__ . "/Models");
+        $config = Setup::createAnnotationMetadataConfiguration($paths, true);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Doctrine\ORM\Tools\Setup' in method 'setUp'.](https://www.codacy.com/app/Managlea/EntityManager/file/2313601649/issues/source?bid=3149133&fileBranchId=3199776#l32)